src/xo-app/home/index.css

```diff
@@ -40,7 +40,6 @@
 .itemExpanded {
   color: #999;
   font-size: 1em;
-  overflow: hidden;
```

Comment:
I need this so the dropdown menu isn't truncated when it's open.
I removed it and tested it. And I couldn't find why it was added in the first place.